tests/test_db.py

```diff
@@ -147,12 +147,12 @@ def testNestedTransactions(self):
     def testPooling(self):
         # can't test pooling if DBUtils is not installed
         try:
-            import DBUtils  # noqa
+            import dbutils  # noqa
```

Comment:
Please be specific about which error(s) we want flake8 to ignore as discussed at
https://flake8.pycqa.org/en/3.9.2/user/violations.html?highlight=noqa#in-line-ignoring-errors